docs/source/quickstart-aws.rst

```diff
@@ -0,0 +1,179 @@
+
+.. code-block:: shell
+
+    # Environment variable
+    export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
+    export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+
+.. code-block:: shell
+
+    # Check whether an AWS configuration file already exists
+    [ -f ~/.aws/config ] || echo "AWS configuration file already exists!"
+
+    # If not, enter the following information in it
+    [ -d ~/.aws ] || mkdir ~/.aws
+    echo '[default]' > ~/.aws/config
+    echo 'aws_access_key_id = <YOUR_ACCESS_KEY_ID>' >> ~/.aws/config
+    echo 'aws_secret_access_key = <YOUR_SECRET_ACCESS_KEY>' >> ~/.aws/config
+
+
+Configure it
+------------
+
+The minimal configuration requires: 
+
+#. :ref:`AWS region <elb_aws_region>` to run ElasticBLAST on (``us-east-1`` recommended, see :ref:`AWS configuration <aws_conf>` for additional details),
+#. :ref:`query sequences <elb_queries>` in a single file or tarball, 
+
+#. a :ref:`cloud storage bucket for results <elb_results_bucket>`. This value must start with ``s3://``.
+
+#. basic BLAST parameters (:ref:`program <elb_blast_program>` and :ref:`database <elb_db>`), and
+
+#. :ref:`elb_num_nodes` to start.
+
+
+
+They can be provided on a standard ini configuration file, e.g.:
+
+.. code-block::
+    :name: minimal-config
+    :linenos:
+
+    [cloud-provider]
+    aws-region = us-east-1
+
+    [cluster]
+    machine-type = m5.8xlarge
+    num-nodes = 1
+
+    [blast]
+    program = blastn
+    db = pdbnt
+    queries = s3://elasticblast-test/queries/MANE.GRCh38.v0.8.select_refseq_rna.fna 
+    results-bucket = ${YOUR_RESULTS_BUCKET}
+
+See :ref:`configuration` for details on all the configuration parameters.
+
+Run it!
+-------
+
+.. code-block:: bash
+
+    ./elastic-blast submit --cfg ${CONFIG_FILE} --loglevel DEBUG
+
+**NOTE: currently you can only have one ElasticBLAST search running at a time**.
+
+
+Monitor progress
+----------------
+To check on the progress of the search, inspect the logfile
+(``elastic-blast.log`` by default) and/or run the command below:
+
+.. code-block:: bash
+    :name: status
+
+    ./elastic-blast status --cfg ${CONFIG_FILE} --loglevel DEBUG
+
+The status command will not return proper results until the submit command has finished.
+
+You can also visit the web intefaces for 
+`CloudFormation <https://console.aws.amazon.com/cloudformation/>`_ and
+`Batch <https://console.aws.amazon.com/batch/>`_ 
+to monitor the progress of your cloud resource creation and jobs respectively.
+
+Problems? Search taking too long? Please see :ref:`support`.
+
+Get results
+-----------
+
+Run the command below to download the results.
+
+**Note**: this command requires the `AWS CLI SDK <https://aws.amazon.com/cli/>`_.
+
+.. code-block:: bash
+
+    aws s3 cp ${YOUR_RESULTS_BUCKET}/*.out.gz . 
+
+Clean up
+--------
+This step is **critical**, please do not omit it, even if you ran Ctrl-C when
+starting ElasticBLAST. If you do not clean up your cloud resources, you will accrue charges from
```

Comment:
The bit about charges is not true for AWS.

Reply:
Thanks for the feedback. Please see 2.a. here: https://confluence.ncbi.nlm.nih.gov/display/EB/2020-10-13+Backlog+grooming